test/mind_the_gapp_web/controllers/arrival_time_controller_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule MindTheGappWeb.ArrivalTimeControllerTest do
+  use MindTheGappWeb.ConnCase, async: true
+  use Timex
+
+  setup_all :create_fake_arrival_time_data
+
+  def create_fake_arrival_time_data(_context) do
+    clear_database()
+    generate_fake_arrival_data()
+    :ok
+  end
+
+  def clear_database() do
+    MindTheGapp.Repo.delete_all(MindTheGapp.ArrivalTime)
+  end
+
+  def generate_fake_arrival_data() do
+    starting_date = DateTime.utc_now
+
+     1..50
+     |> Enum.reduce(starting_date, &persist_arrival_time/2)
+  end
+
+  def persist_arrival_time(_x, date_time) do
+    actual_time = DateTime.add(date_time, Enum.random(-10..10) * 60)
+
+     %MindTheGapp.ArrivalTime{estimated: date_time, actual: actual_time}
+     |> MindTheGapp.Repo.insert!()
+
+    date_time |> DateTime.add(3600 * -6)
+  end
+  describe "Arrival Time Controller" do
+    test "GET / returns a JSON object", %{conn: conn} do
+      conn = get(conn, "/")
+      assert json_response(conn, 200)
+    end
```

Comment:
I might bring in the next and last test here as well. Essentially this is the happy path test and we can assert that we got a 200 and the data matches what we expect.
```suggestion
    test "GET / returns the mean arrival", %{conn: conn} do
      response = get(conn, "/") |> json_responsee(200)
      assert [
        %{day: "Today", meanArrivalDifference: _average},
        %{day: "Yesterday", meanArrivalDifference: _average}
        | _rest
      ] = response
    end
```